src/Cake.Common/Tools/SignTool/SignToolSignSettings.cs

```diff
@@ -33,5 +33,10 @@ public sealed class SignToolSignSettings : ToolSettings
         /// Gets or sets the signed content's description.
         /// </summary>
         public string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the signed content's expanded description URL.
+        /// </summary>
+        public Uri DescriptionUri { get; set; }
```

Comment:
@dbertram @patriksvensson @devlead should this be a Uri?  Arguably, yes, it is a URI, but at the time that Settings class knows about it, it is a string, taken from the command line.  In the GitVersion and GitLink aliases, any URI's were accepted as a string, and then passed to the underlying tool.  I don't think we need to complicate the Cake wrapper, unless we are doing some sort of validation of the URI, in terms of scheme, domain, etc, in which case, converting to a URI would make sense.

Thoughts?